src/DIRAC/__init__.py

```diff
@@ -153,6 +153,22 @@ def _computeRootPath(rootPath):
 from DIRAC.ConfigurationSystem.Client.Config import gConfig
 
 
+def initialize():
```

Comment:
I think there are some arguments that would be useful for this. Perhaps:

```suggestion
def initialize(*, log_level=None, auth_required=True, cert=None, key=None):
```

(The `(*, ` makes them [keyword-only arguments](https://www.python.org/dev/peps/pep-3102/).)

Reply:
Also I'd be tempted to call this `init` rather than `initialize`, but I might be biased by the presence of the `z` 😉

Reply:
>I think there are some arguments that would be useful for this

I personally do not think it is worth burdening this method with arguments, at least for now. But it may make sense whether you could do the appropriate PR or issue, if you have an idea of what should be there.